### ISSUE TITLE
Nothing works on Ubuntu

### ISSUE THREAD

**Issue by WinteryFox** [flozz/rivalcfg#28]
No matter what option I try to change nothing works at all.

```
winteryfox@winteryfox:~$ rivalcfg -l
SteelSeries Rival                             1038:1384   
SteelSeries Rival 100                         1038:1702   
SteelSeries Rival 300                         1038:1710   (plugged)
SteelSeries Rival 300 CS:GO Fade Edition      1038:1394

winteryfox@winteryfox:~$ lsusb
Bus 002 Device 002: ID 8087:8000 Intel Corp. 
Bus 002 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 001 Device 002: ID 8087:8008 Intel Corp. 
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 002: ID 1038:1607 SteelSeries ApS 
Bus 003 Device 003: ID 1038:1710 SteelSeries ApS 
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

winteryfox@winteryfox:~$ dmesg | grep -C 5 hidraw
[    1.235909] sd 1:0:0:0: [sdb] Attached SCSI disk
[    1.239538] usb 3-9: New USB device found, idVendor=1038, idProduct=1607
[    1.239540] usb 3-9: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    1.239540] usb 3-9: Product: SteelSeries Apex M500 Gaming Keyboard
[    1.239541] usb 3-9: Manufacturer: SteelSeries
[    1.241759] hidraw: raw HID events driver (C) Jiri Kosina
[    1.242964] usb 2-1: New USB device found, idVendor=8087, idProduct=8000
[    1.242965] usb 2-1: New USB device strings: Mfr=0, Product=0, SerialNumber=0
[    1.243107] hub 2-1:1.0: USB hub found
[    1.243213] hub 2-1:1.0: 6 ports detected
[    1.243730] usbcore: registered new interface driver usbhid
[    1.243731] usbhid: USB HID core driver
[    1.245018] input: SteelSeries SteelSeries Apex M500 Gaming Keyboard as /devices/pci0000:00/0000:00:14.0/usb3/3-9/3-9:1.0/0003:1038:1607.0001/input/input2
[    1.246154]  sda: sda1 sda2 < sda5 sda6 sda7 >
[    1.246434] sd 0:0:0:0: [sda] Attached SCSI disk
[    1.302688] hid-generic 0003:1038:1607.0001: input,hidraw0: USB HID v1.11 Keyboard [SteelSeries SteelSeries Apex M500 Gaming Keyboard] on usb-0000:00:14.0-9/input0
[    1.302911] hid-generic 0003:1038:1607.0002: hiddev0,hidraw1: USB HID v1.11 Device [SteelSeries SteelSeries Apex M500 Gaming Keyboard] on usb-0000:00:14.0-9/input1
[    1.303230] input: SteelSeries SteelSeries Apex M500 Gaming Keyboard as /devices/pci0000:00/0000:00:14.0/usb3/3-9/3-9:1.2/0003:1038:1607.0003/input/input3
[    1.358594] usb 3-10: new full-speed USB device number 3 using xhci_hcd
[    1.362685] hid-generic 0003:1038:1607.0003: input,hidraw2: USB HID v1.11 Device [SteelSeries SteelSeries Apex M500 Gaming Keyboard] on usb-0000:00:14.0-9/input2
[    1.362893] input: SteelSeries SteelSeries Apex M500 Gaming Keyboard as /devices/pci0000:00/0000:00:14.0/usb3/3-9/3-9:1.3/0003:1038:1607.0004/input/input4                                                                                                               
[    1.422700] hid-generic 0003:1038:1607.0004: input,hidraw3: USB HID v1.11 Keyboard [SteelSeries SteelSeries Apex M500 Gaming Keyboard] on usb-0000:00:14.0-9/input3                                                                                                      
[    1.499320] usb 3-10: New USB device found, idVendor=1038, idProduct=1710                                                          
[    1.499321] usb 3-10: New USB device strings: Mfr=1, Product=2, SerialNumber=0                                                     
[    1.499321] usb 3-10: Product: SteelSeries Rival 300 Gaming Mouse                                                                  
[    1.499322] usb 3-10: Manufacturer: SteelSeries                                                                                    
[    1.500144] hid-generic 0003:1038:1710.0005: hiddev0,hidraw4: USB HID v1.11 Device [SteelSeries SteelSeries Rival 300 Gaming Mouse] on usb-0000:00:14.0-10/input0                                                                                                        
[    1.500554] input: SteelSeries SteelSeries Rival 300 Gaming Mouse as /devices/pci0000:00/0000:00:14.0/usb3/3-10/3-10:1.1/0003:1038:1710.0006/input/input5                                                                                                                
[    1.500675] hid-generic 0003:1038:1710.0006: input,hidraw5: USB HID v1.11 Mouse [SteelSeries SteelSeries Rival 300 Gaming Mouse] on usb-0000:00:14.0-10/input1                                                                                                           
[    1.501424] input: SteelSeries SteelSeries Rival 300 Gaming Mouse as /devices/pci0000:00/0000:00:14.0/usb3/3-10/3-10:1.2/0003:1038:1710.0007/input/input6                                                                                                                
[    1.558694] hid-generic 0003:1038:1710.0007: input,hidraw6: USB HID v1.11 Keyboard [SteelSeries SteelSeries Rival 300 Gaming Mouse] on usb-0000:00:14.0-10/input2                                                                                                        
[    1.730586] tsc: Refined TSC clocksource calibration: 3399.997 MHz                                                                 
[    1.730590] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x31024bd20a6, max_idle_ns: 440795346804 ns                     
[    1.773578] random: fast init done                                                                                                 
[    2.754661] clocksource: Switched to clocksource tsc                                                                               
[    6.973529] EXT4-fs (sda5): mounted filesystem with ordered data mode. Opts: (null) ```

**Comment by flozz** [comment]
Hello,

Which version of Ubuntu you have?

I think this is related to #25 and #5 but I need more informations to solve this as I do not own this mouse :(

**Comment by Thiblizz** [comment]
@flozz, I just got my Rival 500, feel free to send ,me your adress on my e-mail, and I will give you my Rival 300 (brand new).

I will have to go for a Colissimo :).

**Comment by flozz** [comment]
I sent you my address by email, thank you for your help! :D

**Comment by WinteryFox** [comment]
its Kubuntu 17.04, if you need any more info just post the commands here and I'll get it to you

**Comment by flozz** [comment]
I think we will find nothing abnormal in the commands output on the OS, I will wait for the @Thiblizz's Rival 300 to be able to make some test on the mouse itself as it seems that something changed with latest firmware (see #5 for more info) :)

**Comment by flozz** [comment]
Thanks to the @Thiblizz mouse, I found why it does not worked anymore [explanations here](https://github.com/flozz/rivalcfg/issues/25#issuecomment-302195070).

**A fix will be available soon :)**

**Comment by flozz** [comment]
I just released **rivalcfg 2.5.2** and it should work now. I close this issue, please comment in #25 if it does not work for you :)
